scripts/pubmedextract.py

```diff
@@ -0,0 +1,58 @@
+"""
+Extract a set of doc ids from the pubmed xml files.
+"""
+import argparse
+import glob
+import gzip
+import multiprocessing
+import os
+from functools import partial
+from multiprocessing import Pool
+
+import sys
+from lxml import etree
+
+
+def parse_pubmeds(pmids: list, file: str) -> str:
+    """
+
+    :param pmids:
+    :param file:
+    :return:
+    """
+    data = """<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE PubmedArticleSet SYSTEM "http://dtd.nlm.nih.gov/ncbi/pubmed/out/pubmed_170101.dtd">
+<PubmedArticleSet>  
+{}
+</PubmedArticleSet> 
+"""
+    print(file)
+    decompressed_file = gzip.GzipFile(file, mode='rb')
+    tree = etree.parse(decompressed_file)
+    root = tree.getroot()
+    for node in root.findall('PubmedArticle'):
+        pmid = node.find('MedlineCitation').find('PMID').text
+        if pmid in pmids:
+            print(pmid)
+            file_data = data.format(
+                etree.tostring(node, encoding='unicode', method='xml', pretty_print=True))
+            with open('/datadrive2/pubmed_filter/{}.xml'.format(pmid), 'w') as f:
```

Comment:
absolute path?

Reply:
Nice spotting!

Reply:
thanks

Reply:
@leifos lol maybe wait for @hscells to fix the path before merging 💃